packages/analytics/package.json

```diff
@@ -5,14 +5,14 @@
 	"module": "dist/ag.common-analytics.esm.js",
 	"license": "MIT",
 	"peerDependencies": {
-		"@ag.ds-next/react": "^1.13.0",
+		"@ag.ds-next/react": "^1.18.0",
```

Comment:
The packages you've touched here will also need to be bumped. Just add a changeset saying something like "Upgraded AgDS peer pendencies"  etc